link_linux.go

```diff
@@ -1514,6 +1515,16 @@ func linkSubscribeAt(newNs, curNs netns.NsHandle, ch chan<- LinkUpdate, done <-c
 			s.Close()
 		}()
 	}
+	if list {
```

Comment:
Ideally, shouldn't the dump request be performed after spawning the receive go routine ?